### ISSUE TITLE
Executing Python keybinding code defined inside asterisks doesn't work

### ISSUE THREAD

**Issue by Lykdraft** [kmatheussen/radium#1099]
Yeah, looks like a lot changed in the last time. :)

We worked out the stuff below some time ago, but they don't work anymore.
Really need to get those working again. Pre-Roll is utterly important for my editing.

Best,
tL.

```
EXTRA_L UPARROW: *
        ra._start_playing_line = ra.currentLine()
        ra.cursorUp(10)
        ra.playBlockFromCurrent()
*

EXTRA_L DOWNARROW: *
        ra.playStop()
        ra.setCurrentLine(ra._start_playing_line)
*

EXTRA_L RIGHTARROW: *
        ra._start_playing_line = ra.currentLine()
        ra.cursorUp(10)
        ra.playBlockFromCurrent()
*

EXTRA_L LEFTARROW: *
        ra.playStop()
        ra.setCurrentLine(ra._start_playing_line)
*
```

**Comment by kmatheussen** [comment]
This is a bug. Seems like executing code inside asterisks doesn't work. (I changed your code BTW, I guess something went wrong when copying and pasting)

As a workaround you can use this instead:
```
EXTRA_L UPARROW: ra.evalPython "ra._start_playing_line = ra.currentLine();       ra.cursorUp(10);        ra.playBlockFromCurrent();"

EXTRA_L DOWNARROW: ra.evalPython "ra.playStop(); ra.setCurrentLine(ra._start_playing_line)"

EXTRA_L RIGHTARROW: ra.evalPython "ra._start_playing_line = ra.currentLine();      ra.cursorUp(10);        ra.playBlockFromCurrent()"

EXTRA_L LEFTARROW: ra.evalPython "ra.playStop();     ra.setCurrentLine(ra._start_playing_line)"
```

**Comment by kmatheussen** [comment]
(don't close the ticket BTW)

**Comment by kmatheussen** [comment]
Fixed in 5.4.4
